Resources/Prototypes/_StarLight/Roles/salaries.yml

```diff
@@ -18,11 +18,11 @@
     StationEngineer: 55
     TechnicalAssistant: 23
 
-    Warden: 80
+    Warden: 90
     Detective: 80
-    SecurityOfficer: 50
+    Brigmedic: 70
+    SecurityOfficer: 55
     SecurityCadet: 23
-    Brigmedic: 68
```

Comment:
Why?

Reply:
increased salary a little for warden, because its literally the hardest and the most stressful role on the station, no head role comes close.

Changed salary for officer to 55 because in other departments we had 23 for intern role and 55 for regular role, sec was the only one department where it was set to 50

And brigmedic... just wanted to round it, to be honest

Reply:
> increased salary a little for warden, because its literally the hardest and the most stressful role on the station, no head role comes close.
> 
> Changed salary for officer to 55 because in other departments we had 23 for intern role and 55 for regular role, sec was the only one department where it was set to 50
> 
> And brigmedic... just wanted to round it, to be honest

Salaries are made on purpose so that players have to suffer with change and exchange it for trinkets or give it to others for needs

Reply:
of course. and my changes were made based on 2 reasons:
- how much nt rp wise values employee (because even if cap doesnt do shit, he still has to get the biggest salary on station)
- and how hard it is actually for player himself.

Based on those reasons i was pretty sure that warden was underpaid as hell and i couldnt let officer have lesser salary than engineer and doctor